src/tests/multihost/admultidomain/conftest.py

```diff
@@ -182,7 +182,8 @@ def setup_session(request, session_multihost, create_testdir):
         master.update_resolv_conf(session_multihost.ad[1].ip)
     client.client_install_pkgs()
     client.update_resolv_conf(session_multihost.ad[1].ip)
-    client.clear_sssd_cache()
+    # Sssd is not configured yet so we do not start it here.
+    client.clear_sssd_cache(start=False)
```

Comment:
Is this still needed? If so, would it make sense to have this in a separate commit?

Reply:
Yes this is needed. I do not want to spend any more time on this.